doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,56 @@
+.. role:: html(raw)
+   :format: html
+
+.. _embeddings:
+
+Quantum embeddings
+===================
+
+A *Quantum Embedding* represents classical data as quantum states in a Hilbert space, i.e., *a quantum feature map*. An embedding takes a classical datapoint :math:`x` and translates it into a set of gate parameters in a quantum circuit, creating a quantum state :math:`\mid \psi_x \rangle`. This process is a crucial part of designing quantum algorithms and affects their computational power - for more details, see :cite:`Mariabook2018` and :cite:`schuld2018quantum`. 
+
+Let's consider classical input data consisting of :math:`M` examples, with :math:`N` features each: 
+
+.. math:: \mathcal{D}=\{x^{(1)}, \ldots, x^{(M)}\}
+
+where :math:`x^{(i)}` is a :math:`N` dimensional vector for :math:`i=1,\ldots,M`. To embed this data into :math:`n` quantum subsystems (:math:`n` qubits *or* :math:`n` qumodes for discrete and continuous variable quantum computing, respectively), we can use various embedding techniques, some of which are explained briefly below. 
+
+
+Basis Embedding
+^^^^^^^^^^^^^^^^^^^^
+
+For the basis embedding, classical data has to be in the form of binary strings. The embedded quantum state is the bit-wise translation of a binary string to the corresponding states of the quantum subsystems. For example, :math:`x=1001` is represented by the 4-qubit quantum state :math:`\mid 1001 \rangle`. Hence, one bit of classical information is represented by one quantum subsystem.
+
+Let's consider the classical dataset :math:`\mathcal{D}` mentioned above. For basis embedding, each example has to be a N-bit binary string; :math:`x^{(m)}=(b_1,\ldots,b_N)` with :math:`b_i \in \{0,1\}` for :math:`i=1,\ldots,N`. Assuming all features are repesented with unit binary precision (one bit), each input example :math:`x^{(m)}` can be directly mapped to the quantum state :math:`\mid x^{(m)}\rangle`. **This means that the number of quantum subsystems,** :math:`\bm{n}` **, must be at least equal to** :math:`\bm{N}`. The input data can be represented in the computational basis as
```

Comment:
Still, would try to make clear that the embedding is only the rule 01 <-> |01>, which is different from entire dataset.

```suggestion
Let's consider the classical dataset :math:`\mathcal{D}` mentioned above. For basis embedding, each example has to be a N-bit binary string; :math:`x^{(m)}=(b_1,\ldots,b_N)` with :math:`b_i \in \{0,1\}` for :math:`i=1,\ldots,N`. Assuming all features are repesented with unit binary precision (one bit), each input example :math:`x^{(m)}` can be directly mapped to the quantum state :math:`\mid x^{(m)}\rangle`. **This means that the number of quantum subsystems,** :math:`\bm{n}` **, must be at least equal to** :math:`\bm{N}`. An entire dataset can be represented in superpositions of computational basis states as
```